src/client/swagger/managedFunctions.yml

```diff
@@ -5,7 +5,7 @@ info:
 servers:
   - url: ''
 paths:
-  '/api/v2private/functions':
+  '/api/v2/functions':
```

Comment:
I think this route should be paginated. So the user can provide the limit and offset but those parameters should be optional

Reply:
good call thanks, I'll add it to /runs as well.